lib/active_admin/resource/routes.rb

```diff
@@ -36,7 +40,7 @@ def initialize(resource)
         def collection_path(params)
           route_name = route_name(
             resource.resources_configuration[:self][:route_collection_name],
-            (resource.route_uncountable? ? 'index_path' : 'path')
+            suffix: (resource.route_uncountable? ? 'index_path' : 'path')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.